Examples/Python/src/Utilities.cpp

```diff
@@ -0,0 +1,62 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Plugins/Python/Utilities.hpp"
+
+#include "ActsExamples/Utilities/MeasurementMapSelector.hpp"
+#include "ActsExamples/Utilities/ProtoTracksToTracks.hpp"
+#include "ActsExamples/Utilities/PrototracksToSeeds.hpp"
+#include "ActsExamples/Utilities/SeedsToPrototracks.hpp"
+#include "ActsExamples/Utilities/TracksToParameters.hpp"
+#include "ActsExamples/Utilities/TracksToTrajectories.hpp"
+#include "ActsExamples/Utilities/TrajectoriesToPrototracks.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+namespace py = pybind11;
+
+using namespace ActsExamples;
+using namespace Acts;
+
+namespace Acts::Python {
+
+void addUtilities(Context& ctx) {
```

Comment:
Maybe something like track conversion utilities to make it a bit clearer what the contents are?

Reply:
It is named after the folder where the algorithms reside in (`Examples/Algorithms/Utilities`), so I would leave it like that...